app/styles/_action-chip.less

```diff
@@ -0,0 +1,39 @@
+.action-chip {
+  margin: 0 5px 2px 0;
+  font-size: 12px;
+  display: flex;
+  flex-direction: row;
+  .item {
+    padding: .2em .6em .3em;
+    // explicit radius as is possible 1 node will receive all rules
+    &:first-child {
+      border-top-left-radius: 2px;
+      border-bottom-left-radius: 2px;
+    }
+    &:last-child {
+      border-top-right-radius: 2px;
+      border-bottom-right-radius: 2px;
+    }
+    &:nth-child(3) {
```

Comment:
i'd rather avoid using child selectors on things that are going to appear a bunch of times on the page, can we be more declarative with classnames on the sub-elements of the action-chip